.gitignore

```diff
@@ -0,0 +1,12 @@
+*.gem
+Gemfile.lock
```

Comment:
I'm still not sure if we should gitignore the `Gemfile.lock`... But this was part of the default output from `fastlane` so I decided to leave it for now.